packages/webapp/src/delegates/components/my-delegation.tsx

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import { useQuery } from "react-query";
+
+import {
+    useMemberStats,
+    useMemberListByAccountNames,
+    queryMembers,
+} from "_app";
+import { LoadingContainer } from "_app/ui";
+import { DelegateChip, ElectionState } from "elections";
+import { MembersGrid } from "members";
+import { EdenMember, MemberData } from "members/interfaces";
+
+import { ErrorLoadingDelegation } from "./statuses";
+import { LevelHeading } from "./level-heading";
+import { MyDelegationArrow } from "./arrow-container";
+
+interface Props {
+    electionState?: ElectionState;
+    members: MemberData[];
+    myDelegation: EdenMember[];
+}
+
+export const MyDelegation = ({
+    electionState,
+    members,
+    myDelegation,
+}: Props) => (
+    <>
+        <LowerDelegates myDelegation={myDelegation} members={members} />
+        <ChiefDelegates electionState={electionState} />
+    </>
+);
+
+export default MyDelegation;
+
+const LowerDelegates = ({ members, myDelegation }: Props) => {
+    const { data: membersStats, isLoading, isError } = useMemberStats();
+
+    if (isLoading) return <LoadingContainer />;
+    if (isError || !membersStats) return <ErrorLoadingDelegation />;
+
+    const heightOfDelegationWithoutChiefs = membersStats.ranks.length - 2;
```

Comment:
This assumes a post-election state. Either that or it won't consider the top 2 levels prior to end of election, no?

Reply:
Right. Results on the "My Delegation" page do not show up during an election.